bundles/org.openhab.binding.haywardomnilogic/src/main/java/org/openhab/binding/haywardomnilogic/internal/handler/HaywardBridgeHandler.java

```diff
@@ -301,6 +323,14 @@ public synchronized boolean getTelemetryData() throws HaywardException, Interrup
 
         String xmlResponse = httpXmlResponse(urlParameters);
 
+        // Debug: Inject xml file for testing
```

Comment:
You could take a look at unit test cases. There are several other bindings, which test their code by using real world XML files. So, this could be removed from the production code.

Reply:
Not finding any documentation on xml test cases.  Can you reference an example binding?  Thx.

Reply:
Sure, the deutschebahn binding does extensive tests on XML content, which is received from their XML-API. See https://github.com/openhab/openhab-addons/tree/main/bundles/org.openhab.binding.deutschebahn/src/test

Reply:
This is good example of code only used by the developer.  How about I stash this and get this merged.  It will be a while before I have time to dig into the XML test code.

Reply:
OK